MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -307,12 +307,22 @@ public void Present()
             PlatformPresent();
         }
 
-        /*
+#if WINDOWS && DIRECTX
         public void Present(Rectangle? sourceRectangle, Rectangle? destinationRectangle, IntPtr overrideWindowHandle)
         {
-            throw new NotImplementedException();
+            // TODO: How to handle sourceRectangle and destinationRectangle?
+            SwapChainRenderTarget scrt;
+            if (!_overrideWindows.TryGetValue(overrideWindowHandle, out scrt))
+            {
+                scrt = new SwapChainRenderTarget(this, overrideWindowHandle, PresentationParameters.BackBufferWidth, PresentationParameters.BackBufferHeight);

```

Comment:
I am not sure this is the right way to do this...  in fact It seems broken to me.

XNA's Present(IntPtr) was designed around the DX9 Present(HWND)...

 https://msdn.microsoft.com/en-us/library/windows/desktop/bb174423(v=vs.85).aspx

In that method the behavior was that the content of device's default back buffer was copied to the referenced HWND.

You're override here creates a new swap chain which has its own back buffer.  Worse it looks to me the `SwapChainRenderTarget` is always empty as no one can ever call `SetRenderTarget()` with it.